app/services/apollo.ts

```diff
@@ -80,6 +89,20 @@ export default class Apollo extends ApolloService {
       };
     });
   }
+
+  private createErrorLink() {
+    return onError(({graphQLErrors, networkError}) => {
+      if (graphQLErrors) {
+        graphQLErrors.forEach(({message, locations, path}) => {
+          Sentry.captureMessage(`${message}\n${locations}\n${path}`);
```

Comment:
Should we add "labels" to know what each line represents?

```js
Sentry.captureMessage(`
  Message: ${message} \n
  Location: ${locations} \n
  Path: ${path}
`);
```

Reply:
Sure sure